files/rh-docker/help.1

```diff
@@ -0,0 +1,74 @@
+.PP
+%
+.BR NEXUS (1)
+Container Image Pages
+% Sonatype
+% December 15, 2017
```

Comment:
date seems suspect given this is brand new?

Reply:
maybe it needs to be regenerated after the md file is updated?